src/apic-extension/azext_apic_extension/aaz/latest/apic/integration/__cmd_group.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
you can ignore the files under the `aaz` folder, which is generated automatically